### PR TITLE
Fixed a link to Roadmap & improved Readme

### PR DESCRIPTION
**Pull request by bitcrazed**
Fixed a link to Roadmap & improved Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,86 @@
...
-  * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
+Other related repositories include:
+
+* [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
 
-## Installation
+## Installing & running Windows Terminal
 
-_(Note: in order to run the Windows Terminal, you'll need to be running at least Windows build 18362 or higher.)_
+> 👉 Note: To run the Windows Terminal you'll need to be running at  Windows 10 build 1903 (build 18362) or later
```
Comment (DHowett-MSFT) [inline README.md]:
```suggestion
> 👉 Note: To run the Windows Terminal you'll need to be running Windows 10 1903 (build 18362) or later
```

File: README.md [hunk 2]
```diff
@@ -1,75 +1,86 @@
...
 
-### Microsoft Store
+### Install Windows Terminal via the Microsoft Store
```
Comment (DHowett-MSFT) [inline README.md]:
```suggestion
### From the Microsoft Store
```

File: README.md [hunk 3]
```diff
@@ -1,75 +1,86 @@
...
 
-Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated. Or, feel free to side-load [releases](https://github.com/microsoft/terminal/releases) from GitHub, but note they won't auto-update.
+Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated:
 
 <a href='//www.microsoft.com/store/apps/9n0dx20hk701?cid=storebadge&ocid=badge'><img src='https://assets.windowsphone.com/85864462-9c82-451e-9355-a3d5f874397a/English_get-it-from-MS_InvariantCulture_Default.png' alt='English badge' width="284" height="104" style='width: 284px; height: 104px;'/></a>
 
-### Chocolatey (Unofficial)
+### Download Windows Terminal from this repo's releases
```
Comment (DHowett-MSFT) [inline README.md]:
```suggestion
### From this repository
```

Comment (DHowett-MSFT) [reply README.md]: (nit: subheadings can continue the thought from the parent heading)

File: README.md [hunk 4]
```diff
@@ -1,75 +1,86 @@
...
+Every public release of Windows Terminal is avaiable for manual download from [this repo's Releases page](https://github.com/microsoft/terminal/releases)
+
+> ⚠ Note: If you manually install Terminal from [releases](https://github.com/microsoft/terminal/releases) they won't auto-update so be sure to regularly install the latest Terminal release to receive all the latest fixes and improvements!
+
+### Install Windows Terminal via Chocolatey (unofficial)
```
Comment (DHowett-MSFT) [inline README.md]:
```suggestion
### Via Chocolatey (unofficial)
```

File: README.md [hunk 5]
```diff
@@ -1,75 +1,86 @@
...
 
 Since assuming ownership of the Windows command-line in 2014, the team has added several new features to the Console, including window transparency, line-based selection, support for [ANSI / Virtual Terminal sequences](https://en.wikipedia.org/wiki/ANSI_escape_code), [24-bit color](https://devblogs.microsoft.com/commandline/24-bit-color-in-the-windows-console/), a [Pseudoconsole ("ConPTY")](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/), and more.
 
-However, because the Console's primary goal is to maintain backward compatibility, we've been unable to add many of the features the community has been asking for, and which we've been wanting to add for the last several years--like tabs!
+However, because the Console's primary goal is to maintain backward compatibility, we've been unable to add many of the features the community has been asking for (and which we've been wanting to add) for the last several years - like tabs, unicode text, emoji, etc!
```
Comment (DHowett-MSFT) [inline README.md]: nit: no need for "like" and "etc" in the same sentence. They both imply "i am giving an incomplete list"

Comment (bitcrazed) [reply README.md]: Good point. Corrected.

**Comment by bitcrazed** [comment]
Thanks for the edits & suggestions all.

**Pull request merged**